src/fsutils.ts

```diff
@@ -15,3 +16,14 @@ export function dirExistsSync(path: string): boolean {
     return false;
   }
 }
+
+export function readFile(path: string): string {
+  try {
+    return readFileSync(path).toString();
+  } catch (e: any) {
```

Comment:
Either don't give `e` a type or type it as `unknown`. You'll have to do an extra cast/check to get `code`, but it's worth avoid the `any` if possible.

https://google.github.io/styleguide/tsguide.html#any

Reply:
Done.